app/containers/HomeContainer/index.js

```diff
@@ -37,71 +37,71 @@ const Container = styled.div`
     padding: ${props => props.padding}px;
   }
 `;
-const RightContent = styled.div`
-  display: flex;
-  align-self: flex-end;
-`;
+// const RightContent = styled.div`
+//   display: flex;
+//   align-self: flex-end;
+// `;
 export function HomeContainer({
-  dispatchGithubRepos,
-  dispatchClearGithubRepos,
+  dispatchArtist,
+  dispatchClearArtistSearch,
   intl,
-  reposData = {},
-  reposError = null,
-  repoName,
+  artistData = {},
+  artistSearchError = null,
+  artistName,
```

Comment:
Better way is to use default props for this

HomeContainer.defaultProps = {
     artistData : {},
     artistSearchError : null
}